### ISSUE TITLE
Request: Add Back/Forward navigation keys

### ISSUE THREAD

**Issue by equin0x80** [simonsteele/pn#211]
I know this app is not updated any more, but it's still my favourite -- so, just in case Simon changes his mind, or it gets forked:

When working with long documents, it would be great to be able to navigate Back and Forward. For example, I delete a line from the middle, then I jump to the top to copy some text, then I want to go back where I was. Currently the "easiest" way is Ctrl+Z Ctrl+Y (undo redo), because that incidentally puts me back where I was -- but clearly this isn't ideal, and could lead to mistakes.

Suitable shortcut keys would be Ctrl Plus and Ctrl Minus (as in Visual Studio), or Alt+Left and Alt+Right (as in some Web browsers).

**Comment by veganaiZe** [comment]
Have you considered writing an [extension](http://www.pnotepad.org/docs/dev/writing_your_first_extension/)?

**Comment by equin0x80** [comment]
No. Have you?

**Comment by veganaiZe** [comment]
Yeah...  I've tried creating a hello world [python extension for PN](https://www.pnotepad.org/docs/pypn_overview/) (requires installing [python 3.4](https://www.python.org/downloads/release/python-344/) support) but it seemed like it wasn't fully implemented and, it may have just been a coincidence but, it also seemed to make PN buggy (maybe it activated some bunk extension?).  --I managed to get hello world working, though, at least.

If I were to attempt it again I'd probably try out the C++ route.

**Comment by equin0x80** [comment]
Yes, that's exactly why I don't waste my time. I look forward to your excellent text editor, but until then, I will 
 use this one. If you have nothing to say about the actual topic (back/forward keys) then I don't know why you are here, frankly.

**Comment by veganaiZe** [comment]
I'm talking about writing an extension for Programmer's Notepad, not writing a new IDE.
